build-resources

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Perhaps rename the file to .sh

Reply:
It's an executable script. It's a common convention to not have file extensions on executable files. See, e.g. https://askubuntu.com/questions/503127/should-i-save-my-scripts-with-the-sh-extension#503129

(Yeah, I know that many people coming from Windows world don't do this correctly and that's why you so often see `.sh` suffix on exectables too.)

Reply:
or https://google.github.io/styleguide/shell.xml#File_Extensions